data/README.md

```diff
@@ -80,6 +80,40 @@ The return value of `registerStore` is a [Redux-like store object](https://redux
 - `store.dispatch( action: Object )`: Given an action object, calls the registered reducer and updates the state value.
    - _Redux parallel:_ [`dispatch`](https://redux.js.org/api-reference/store#dispatch(action))
 
+### Helpers
```

Comment:
I don't think this section is important enough to place before API reference. Seems like it should be moved below "Data Access and Manipulation".

Reply:
Sure thing. I wasn't sure where to put it.